client/src/modules/uncles/components/TableUncles.vue

```diff
@@ -81,7 +81,7 @@ import AppTableTitle from '@app/core/components/ui/AppTableTitle.vue'
 import TableUnclesRow from '@app/modules/uncles/components/TableUnclesRow.vue'
 import TableUnclesRowLoading from '@app/modules/uncles/components/TableUnclesRowLoading.vue'
 import { Vue, Component, Prop } from 'vue-property-decorator'
-import { UncleSummaryPageExt } from '@app/core/api/apollo/extensions/uncle-summary-page.ext'
+// import { UncleSummaryPageExt } from '@app/core/api/apollo/extensions/uncle-summary-page.ext'
```

Comment:
you can delete this

Reply:
oops! I actually deleted this in the new client folder, and actually pushed this from client, I will remove this